addons/stock_calendar/procurement.py

```diff
@@ -170,8 +170,8 @@ def _get_group(self, cr, uid, orderpoint, context=None):
         calendar_obj = self.pool.get("resource.calendar")
         att_obj = self.pool.get("resource.calendar.attendance")
         group = False
-        context = context or {}
-        context['no_round_hours'] = True
+        new_context = context.copy() or {}
```

Comment:
That will crash if `context` is `None` (which should probably not happen).
I recommend writing this as:
```python
    new_context = dict(context or {})
```
This is more explicit than relying on `copy()` to return a `dict` instead of a `frozendict`...